tests/__init__.py

```diff
@@ -14,10 +14,11 @@
 from tests import test_inout
 from tests import test_literaltypes
 from tests import validator
-from tests import test_ows
```

Comment:
Why is the test_ows disabled?

Reply:
oops ... I checked it on my side and it must have happend during development ... test_ows failed without gdal ... didn't notice.

I've enabled these tests again and they pass (with gdal). 

Should I open a new PR or is this acceptable?

Reply:
leave it as it is I also tested and things where ok, later will patch